connectivity/drivers/cellular/Samsung/S5JS100_RIL/SAMSUNG_S5JS100_RIL.cpp

```diff
@@ -0,0 +1,188 @@
+/*
+ * Copyright (c) 2020, Arm Limited and affiliates.
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#include "SAMSUNG_S5JS100_RIL.h"
+#include "SAMSUNG_S5JS100_RIL_CellularContext.h"
+#include "SAMSUNG_S5JS100_RIL_CellularNetwork.h"
+#include "SAMSUNG_S5JS100_RIL_CellularInformation.h"
+#include "SAMSUNG_S5JS100_RIL_IF.h"
+#include "CellularLog.h"
+#include "ScopedLock.h"
+
+using namespace mbed;
+typedef enum s5js100_cellular_event_status {
```

Comment:
This should be moved to `cellular_connection_status_t `list in `CellularCommon.h` even though this is RIL specific. That way we ensure these numbers stay in sync.

Reply:
This PR is based on mbed os 6 to add RIL interface, the pr-S5JS100-06_RIL is based on mbed os 5, so it not in `CellularCommon.h`